perun-core/src/main/java/cz/metacentrum/perun/core/impl/GroupsManagerImpl.java

```diff
@@ -483,7 +489,7 @@ public boolean isDirectGroupMember(PerunSession sess, Group group, Member member
 	public void removeMember(PerunSession sess, Group group, Member member) throws InternalErrorException, NotGroupMemberException {
```

Comment:
Check on sourceGroupId != null is there, but there is still missing information in javadoc, that member with set sourceGroupId is needed.